scripts/o3de/o3de/repo.py

```diff
@@ -220,7 +219,7 @@ def search_repo(manifest_json_data: dict,
         o3de_object_uris = manifest_json_data['template']
```

Comment:
Actually the wrong key is being looked up here. It should be `templates` with an `s`.
```suggestion
        o3de_object_uris = manifest_json_data['templates']
```